project/src/components/reviews-form/reviews-form.jsx

```diff
@@ -0,0 +1,58 @@
+import { React, useState } from 'react';
+
+function ReviewsForm() {
+  const state = useState('test');
+  let review = state[0];
+  const setReview = state[1];
```

Comment:
```suggestion
  const [review, setReview] = useState('test');
```
Тут вообще что-то странное, тесты какие-то